.github/dependabot.yml

```diff
@@ -8,13 +11,20 @@ updates:
       prefix: ":seedling:"
   labels:
     - "ok-to-test"
+# Go
 - package-ecosystem: "gomod"
   directory: "/"
   schedule:
     interval: "weekly"
-  allow:
-    - dependency-name: "github.com/coredns/corefile-migration"
+  ignore:
```

Comment:
We should also ignore dependencies that need to be bumped also in the Makefile: like Ginkgo, envsubst etc.

Reply:
Should be fine if we merge https://github.com/kubernetes-sigs/cluster-api/pull/7503/files before, correct?

Reply:
Yes, I both hope and think so :)